packages/components/text/src/Text.settings.ts

```diff
@@ -5,9 +5,7 @@ import { IComposeSettings } from '@uifabricshared/foundation-compose';
 export const settings: IComposeSettings<ITextType> = [
   {
     tokens: {
-      fontFamily: 'primary',
-      fontSize: 'medium',
-      fontWeight: 'medium',
+      fontVariant: 'secondaryStandard',
```

Comment:
I would expect body to be the default for all the controls

Reply:
Make sense given the name but this would increase the size from what it is currently (12px -> 14px).

Reply:
Is that what we want?

Reply:
Probably not.  Let's leave it as is until we get official redlines.